tests/integration/test_lists/waives.txt

```diff
@@ -312,13 +311,8 @@ triton_server/test_triton.py::test_python_bls_unit_tests[python-bls-unit-tests]
 triton_server/test_triton.py::test_mistral_ib[mistral-ib] SKIP (https://nvbugs/5477399)
 triton_server/test_triton.py::test_eagle[eagle] SKIP (https://nvbugs/5477378)
 examples/test_mixtral.py::test_llm_mixtral_moe_plugin_lora_4gpus[Mixtral-8x7B-v0.1-chinese-mixtral-lora] SKIP (https://nvbugs/5477421)
-accuracy/test_llm_api_pytorch.py::TestDeepSeekR1::test_nvfp4_multi_gpus[throughput_tp8] SKIP (https://nvbugs/5455140)
```

Comment:
This OOM is a dlcluster infra issue, we think its fixed, and enable it to observe.